domains/fuyukawa.json

```diff
@@ -0,0 +1,10 @@
+{
+  "subdomain": "fuyukawa",
```

Comment:
We dont have that in `is-a.dev`, and this domain **isnt** managed by Open Domains
```suggestion
```